astroquery/wfau/core.py

```diff
@@ -290,10 +291,49 @@ def get_images_async(self, coordinates, *, waveband='all', frame_type='stack',
                                       show_progress=show_progress)
                 for url in image_urls]
 
-    def get_image_list(self, coordinates, *, waveband='all', frame_type='stack',
-                       image_width=1 * u.arcmin, image_height=None,
-                       radius=None, database=None,
-                       programme_id=None, get_query_payload=False):
+    def get_image_list(self, coordinates, *, radius=None, ignore_deprecated=True, **kwargs):
```

Comment:
spell out kwargs, we try to get rid of this way of hiding them so we should not introduce more of these.

Reply:
hmm, ok.  That's a lot of duplicated text, which I don't love, but fine

Reply:
yes, but if it's public API, one would need useful docstrings, too. dumping kwarg out there is not exactly useful. OTOH, if it's an option of not getting slightly overlapping public API then certainly I would be in favour of not duplicating.

Reply:
`get_image_list` is more of an internal-pointing method.  It is technically usable, and useful, to users, but `get_image_table` is more useful and I prefer to point people there. I'd actually lean toward making `_get_image_list` private before duplicating its docstring and keyword arguments.  Let me know which you prefer